packages/lesswrong/components/votes/VoteArrowIcon.tsx

```diff
@@ -6,7 +6,6 @@ import ArrowDropUpIcon from '@material-ui/icons/ArrowDropUp';
 import IconButton from '@material-ui/core/IconButton';
 import Transition from 'react-transition-group/Transition';
 import { VoteColor, cssVoteColors } from './voteColors';
-import { isEAForum } from '../../lib/instanceSettings';
```

Comment:
You can ignore the changes in this file, I tried using this component then decided not to, so this is just removing unused code